tests/fixtures/marian

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+"""
+marian test fixture
+
+Do not rely on marian in tests. This mocks marian by outputing the training corpus
```

Comment:
Suggestion: I found this comment a little confusing, as we can now rely on Marian in tests. Maybe renaming it to "Marian can be mocked for tests when the actual binary doesn't need to be run.".